src/api/users/users.post.test.ts

```diff
@@ -42,3 +42,130 @@ describe('User Creation', () => {
     await prisma.user.deleteMany({ where: { email: USER_TO_CREATE.email } });
   });
 });
+
+describe('User Creation Errors', () => {
+  it('should return invalid email error', async () => {
+    const response = await axios.post(
+      'http://localhost:8080/users',
+      {
+        ...USER_TO_CREATE,
+        email: 'invalid-email',
+      },
+      { validateStatus: (status: number) => status === 422 },
```

Comment:
```suggestion
      { validateStatus: () => true },
```

- Como vc já checa ali embaixo, não precisa re-checar aqui.
- E aí considerando isso, acho que vc pode reaproveitar uma função que faz a request e deixar a mais limpa, como se fosse um "client". Exemplo:

```ts
export function sendTestRequest(method: Method, body: any) {
  return axios.request({ url: ''http://localhost:8080/users', method, body }, { validateStatus: () => true })
}
```

Sua utilização fica mais simples:

```ts
const response =  await sedTestRequest('post', { ...USER_TO_CREATE, email: 'invalid-email' });
```

Mais pra frente vai ser mais simples e reaproveitável tbm pra enviar o header de autenticação. Essa função seria o equivalente ao que temos no template como o RequestMaker, só que lá é um pouco mais robusto.

**OBS:** não sei se eu acertei exatamente o modelo correto da `request` do axios pq eu chutei, mas vc conserta ai.